frameworks/spring-boot/src/it/issue-370-probe-properties/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8u171-alpine3.7
```

Comment:
Doesn't the PR fix issue 369, not issue 370? Asking because the directory is named `issue-370-probe-properties`